common/djangoapps/util/date_utils.py

```diff
@@ -62,6 +63,19 @@ def get_time_display(dtime, format_string=None, coerce_tz=None):
         return get_default_time_display(dtime)
 
 
+def get_formatted_time_zone(time_zone, option=None):
+    """
+    Returns a formatted time zone (e.g. 'Asia/Tokyo (JST +0900)') for user account settings time zone drop down
+    """
+    abbr = now().astimezone(timezone(time_zone)).strftime('%Z')
+    offset = now().astimezone(timezone(time_zone)).strftime('%z')
+    if option == 'abbr':
```

Comment:
Why are these options passed in as strings instead of python keyword arguments?  Also, I don't see any calls that actually pass in a value for `option`.